styles/globals.scss

```diff
@@ -1,3 +1,112 @@
+// Introduction
+// Welcome to the CSS file! This is subjectively the most exciting code
+// of the website. Here we bring together the visual style of the website.
+
+// Our CSS is organized using Harry Roberts' (csswizardry.com) Inverted
+// Triangle CSS (ITCSS) organizational approach. This method is mixed with
+// Block Element Modifier (BEM) naming convention for class names throughout
+// the stylesheet.
+
+// Table of Contents:
+// 1. Settings
+// 2. Tools
+// 3. Generic
+// 4. Elements
+// 5. Objects
+// 6. Components
+// 7. Vendors
+// 8. Utilities
+
+// =====================================================================
+// 1. Settings
+//    When using a preprocessor, settings are used to define variable.
+//    This puts variables at the top of the cascade so they are accessible
+//    to all partials further down. No CSS selectors should be generated
+//    by the preprocessor from partials in this section.
+
+//    When not using a preprocessor, the settings area is a good place
+//    to write comments describing the colors and font stacks as a guide.
+//    If using features such as custom properties, those that are global
+//    custom properties should be described in this section.
 @import "./settings/colors";
-@import "./settings/spacing";
+
+// =====================================================================
+// 2. Tools
+//    This section is specifically for preprocessor global mixins and
+//    functions. No CSS should be generated by the preprocessor from
+//    partials in this section.
+
 @import "./tools/functions";
+@import "./tools/mixins";
+
+// =====================================================================
+// 3. Generic
+//    Here we define any generic styles that are not specific to the styles of
+//    the site. This section should include any Reset or Normalize libraries
+//    and any preferential base styles for elements. There shouldn't be any
+//    classes or ids used in this section.
+
+// 4. Elements
+//    Styling for bare HTML elements (like H1, A, etc.). These come with
+//    default styling from the browser so we can redefine them here.
+
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    "Fira Sans",
+    "Droid Sans",
+    "Helvetica Neue",
+    sans-serif;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
This probably got obliterated in conflict resolution, but these styles were removed in an earlier PR, so we should get rid of them here, too.